notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Make sure to install the version corresponding to this tutorial
</pre><p>What are these versions? Perhaps, we can recommend to install the latest (add <code><strong>-U</strong></code><strong> </strong>flag to install the newest versions). We can also probably remove the comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>